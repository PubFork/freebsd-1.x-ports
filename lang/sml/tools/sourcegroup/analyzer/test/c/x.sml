structure A =
  let structure B = struct val x = 3 end
      structure C = struct val y = B.x end
  in
    struct
      val c = C.y
    end
  end
