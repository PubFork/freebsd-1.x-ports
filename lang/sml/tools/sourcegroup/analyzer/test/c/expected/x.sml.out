source sml "c/x.sml"
  export structure A ;

